barretenberg/cpp/src/barretenberg/relations/ecc_vm/ecc_wnaf_relation_impl.hpp

```diff
@@ -210,7 +244,7 @@ void ECCVMWnafRelationImpl<FF>::accumulate(ContainerOverSubrelations& accumulato
     std::get<18>(accumulator) += precompute_select_zero * round;
     std::get<19>(accumulator) += precompute_select_zero * pc;
 
-    // TODO(@zac-williamson #2226)
+    // Optimize(@zac-williamson #2226)
```

Comment:
nit: replace `Optimize` with `NOTE` (that's what we've been using to mark potential optimisations for future - idea is that we don't necessarily want to commit to any optimisations)